test/lib/jdk/test/lib/jfr/EventNames.java

```diff
@@ -192,6 +192,7 @@ public class EventNames {
     public final static String TLSHandshake = PREFIX + "TLSHandshake";
     public final static String X509Certificate = PREFIX + "X509Certificate";
     public final static String X509Validation = PREFIX + "X509Validation";
+    public final static String InitialSecurityProperty = PREFIX + "InitialSecurityProperty";
```

Comment:
Let's unify order of modifiers in this file. Some fields use `public final static` and some `public static final`